modules/is_master/templates/build.sh.erb

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# ----------------------------------------------------------------------------
+
+# Apply configurations
+export FACTER_profile=is_master
+puppet agent -vt
+
+# exit immediately if a command exits with a non-zero status
+set -e
+
+updated_templates=()
+
+copy_to_agent() {
+  cd <%= @distribution_path %>
```

Comment:
AFAIU, the path parameters are relative to the location of the build script. What happens if the user runs the script from elsewhere?

Reply:
the absolute path /usr/local/wso2/wso2is/5.7.0 is set as the distribution_path. Therefore it wont affect if the script runs from elsewhere